Source/DataSources/PointVisualizer.js

```diff
@@ -204,29 +291,77 @@ define([
                     items.set(entity.id, new EntityData(entity));
                 }
             } else {
-                returnPointPrimitive(items.get(entity.id), unusedIndexes);
+                returnPrimitive(items.get(entity.id), unusedPointIndexes, unusedBillboardIndexes);
                 items.remove(entity.id);
             }
         }
 
         for (i = removed.length - 1; i > -1; i--) {
             entity = removed[i];
-            returnPointPrimitive(items.get(entity.id), unusedIndexes);
+            returnPrimitive(items.get(entity.id), unusedPointIndexes, unusedBillboardIndexes);
             items.remove(entity.id);
         }
     };
 
-    function returnPointPrimitive(item, unusedIndexes) {
+    function returnPrimitive(item, unusedPointIndexes, unusedBillboardIndexes) {
         if (defined(item)) {
             var pointPrimitive = item.pointPrimitive;
             if (defined(pointPrimitive)) {
                 item.pointPrimitive = undefined;
                 pointPrimitive.id = undefined;
                 pointPrimitive.show = false;
-                unusedIndexes.push(pointPrimitive._index);
+                unusedPointIndexes.push(pointPrimitive._index);
+                return;
+            }
+            var billboard = item.billboard;
+            if (defined(billboard)) {
+                item.billboard = undefined;
+                billboard.id = undefined;
+                billboard.show = false;
+                unusedBillboardIndexes.push(billboard._index);
             }
         }
     }
 
+    function createCallback(centerAlpha, cssColor, cssOutlineColor, cssOutlineWidth, newPixelSize) {
```

Comment:
Was this copy and pasted from elsewhere?  If so, should this be a `@private` function used in both places?